main.tf

```diff
@@ -45,7 +45,10 @@ resource "azurerm_app_service" "main" {
     }
 
     app_settings = {
+    "BOARD_ID" = "Board123"
     "DOCKER_REGISTRY_SERVER_URL" = "https://index.docker.io"
+    "DOCKER_USER" = "madhuaxp"
+    "OAUTHLIB_INSECURE_TRANSPORT" = "1"
```

Comment:
This still isn't using some of your variables (like CLIENT_ID, CLIENT_SECRET). I'd expect this to look more like..

```
    app_settings = {
    "BOARD_ID" = var.BOARD_ID
    "DOCKER_REGISTRY_SERVER_URL" = "https://index.docker.io"
    "DOCKER_USER" = var.DOCKER_USER
    "CLIENT_ID" = var.CLIENT_ID
...
```